client/pyproject.toml

```diff
@@ -4,3 +4,4 @@ description="AYON Phostoshop addon."
 
 [ayon.runtimeDependencies]
 wsrpc_aiohttp = "^3.1.1" # websocket server
+platformdirs = "*"
```

Comment:
This is not necessary, `platformdirs` are already in `ayon-launcher` https://github.com/ynput/ayon-launcher/blob/develop/pyproject.toml#L35

Reply:
Actually even if they are - technically we should be including it here too. Right @iLLiCiTiT ? Or?